appeals/api/src/server/endpoints/constants.js

```diff
@@ -31,7 +31,7 @@ export const AUDIT_TRAIL_DOCUMENT_UNREDACTED =
 export const AUDIT_TRAIL_DOCUMENT_NO_REDACTION_REQUIRED =
 	'Document {replacement0} (version {replacement1}) marked as requiring no redaction';
 export const AUDIT_TRAIL_DOCUMENT_DATE_CHANGED =
-	'Document {replacement0} (version {replacement1}) received';
+	'Document {replacement0} (version {replacement1}) received date changed';
```

Comment:
should we keep `received`?
e.g. Document 12345 (version 1) received date changed